Tests/Helpers/RuntimeHelpers.swift

```diff
@@ -103,28 +105,34 @@ final class RuntimeHelper {
     }
 
     static let dummyRuntimeMetadata: RuntimeMetadata = {
-        RuntimeMetadata(metaReserved: 1,
-                        runtimeMetadataVersion: 1,
-                        modules: [
-                            ModuleMetadata(name: "A",
-                                           storage: StorageMetadata(prefix: "_A", entries: []),
-                                           calls: [
-                                            FunctionMetadata(name: "B",
-                                                             arguments: [
-                                                                FunctionArgumentMetadata(name: "arg1", type: "bool"),
-                                                                FunctionArgumentMetadata(name: "arg2", type: "u8")
-                                                             ], documentation: [])
-                                           ],
-                                           events: [
-                                            EventMetadata(name: "A",
-                                                          arguments: ["bool", "u8"],
-                                                          documentation: [])
-                                           ],
-                                           constants: [],
-                                           errors: [],
-                                           index: 1)
-                        ],
-                        extrinsic: ExtrinsicMetadata(version: 1,
-                                                     signedExtensions: []))
+        RuntimeMetadata.v1(
```

Comment:
Should we have same for v14? Of course, no time for tests now, but mb it'll be useful to have TODO here?